sematic/api/endpoints/runs.py

```diff
@@ -191,6 +196,21 @@ def _make_cursor(key: str) -> str:
     return base64.urlsafe_b64encode(bytes(key, "utf-8")).decode("utf-8")
 
 
+def _generateSearchPredicate() -> Optional[BooleanClauseList]:
+    search_string, search_predicates = flask.request.args.get("search"), None
+    if search_string is not None:
+        search_predicates = sqlalchemy.or_(
+            Run.name.ilike(f"%{search_string}%"),
+            Run.calculator_path.ilike(f"%{search_string}%"),
+            Run.description.ilike(f"%{search_string}%"),
+            Run.source_code.ilike(f"%{search_string}%"),
```

Comment:
Maybe we should remove search in `description` and `source_code` to make results more relevant. What do you think?

Reply:
Maybe just `source_code`? I don't have a reason why `description` need to be removed.